src/main/java/cz/czechitas/automation/WebDriverProvider.java

```diff
@@ -53,8 +55,21 @@ private static WebDriver createChromeDriver() {
     }
 
     private static WebDriver createEdgeDriver() {
-        EdgeOptions options = new EdgeOptions();
-        options.addArguments("--remote-allow-origins=*");
-        return WebDriverManager.edgedriver().capabilities(options).create();
+        try{
+            EdgeOptions options = new EdgeOptions();
+            options.addArguments("--remote-allow-origins=*");
+            return WebDriverManager.edgedriver().capabilities(options).create();
+        } catch (WebDriverManagerException exception1) {
+            return createSafariDriver();
+        }
+
+    }
+
+    private static WebDriver createSafariDriver(){
```

Comment:
Drzme prosim jednotny code style a dejme tu zavorku na novy radek :]

Do příštího běhu to předělám nějak inteligentněji. Tento vodopád exceptions už přerostl až moc :D

Reply:
Všude kde se dívám do kódu je závorka na stejném řádku, pouze u některých konstruktorů je to ala dotnet. Měl jsem za to, že java jeden tenhle style.. takže mě ten Honzův style tolik nerozhodil.

Reply:
Promin, me tam palila jen ta chybejici mezera mezi zavorkama. Switchoval jsem z jineho projektu, tak jsem neprepl kontext jeste.